qiskit_ibm_runtime/utils/json.py

```diff
@@ -215,7 +215,7 @@ def default(self, obj: Any) -> Any:  # pylint: disable=arguments-differ
         if hasattr(obj, "to_json"):
             return {"__type__": "to_json", "__value__": obj.to_json()}
         if isinstance(obj, QuantumCircuit):
-            kwargs: dict[str, object] = {"use_symengine": bool(optionals.HAS_SYMENGINE)}
+            kwargs: Dict[str, object] = {"use_symengine": bool(optionals.HAS_SYMENGINE)}
```

Comment:
This is just an FYI, but importing `Dict`, `List`, `Union`, `Tuple`... is the older way of typing,  this change makes it consistent with the rest of the file so I think it's fine for this PR, but it might make sense to start using the new typing syntax (which still requires `from __future__ import annotations` in Python 3.8)

Reply:
Thank you, I'll keep that in mind